.env

```diff
@@ -4,6 +4,7 @@
 # Do not commit your values in this file to the repo, but do
 # commit new keys to this example file
 
+FLASK_APP=web
```

Comment:
Unfortunately this will only work if everyone has something like [autoenv](https://github.com/kennethreitz/autoenv) or [direnv](https://direnv.net/) installed.  If we don't have this installed then we need to already know where the web app is before we can load the `.env` file.

Would strongly recommend reverting this change.  Even though it makes me a :sad_panda:.

Reply:
> If python-dotenv is installed, running the flask command will set environment variables defined in the files .env and .flaskenv. This can be used to avoid having to set FLASK_APP manually every time you open a new terminal, and to set configuration using environment variables similar to how some deployment services work.

This is from the official guide. Therefore, if `python-dotenv` is installed (which should be after pip install), then flask will find .env file automatically

Reply:
http://flask.pocoo.org/docs/dev/cli/

Reply:
Woah! This makes me a happy panda! I didn't realize that flask explicitly used python-dotenv if installed!  Awesome!